jekyll/_cci2/use-the-cli-for-config-policy-management.adoc

```diff
@@ -0,0 +1,173 @@
+----
++
+Now, when a pipeline is triggered within your organization, the project's `.circleci/config.yml` will be validated against this policy.
+
+== Update a policy
+
+To illustrate making a change to an existing policy, suppose you made an error when creating the policy above. You realise that configs in your organization are using CircleCI config version `2.0` and you want your policy to reflect this.
+
+. Change the rule definition in your `version.rego` file:
++
+[source,rego]
+----
+{
+    not input.version >= 2.0 # check that version is at least 2.0
+    reason := sprintf("version must be at least 2.0 but got %s", [input.version])
+}
+----
+
+. Push the policy directory containing the updated policy file using the CLI (verify the diff, and choose yes when prompted):
++
+[source,shell]
+----
+circleci-cli policy push ./config-policies --owner-id <your-organization-ID>
+----
+
+== Manage policies with your VCS
+
+CircleCI policies are managed by pushing directories of policies to CircleCI via the CLI. The recommended method for managing your policy directory is by storing them in a repository in your VCS, within your organization. This is how policies are managed internally at CircleCI. Pushing policy bundles is done by triggering CircleCI pipelines.
+
+We recommend creating a bot account for pushing policies, and to use its associated CircleCI personal API token for authentication. For maximum security the token should be stored as an environment variable within a context, and that context should be restricted to groups that are responsible for managing policies. For more information, see the link:/docs/contexts[Using Contexts] page.
+
+=== Set up a config policy management CI pipeline
+
+. Set up repository in your VCS to manage policies.
+
+. Create a directory for your `rego` policy files, for example:
++
+[source,shell]
+----
+mkdir ./config-policies
+----
+
+. Create a `.circleci/config.yml` file for your policies repository, and copy and paste the config example below. This example pushes policies to CircleCI on commits to the `main` branch, and shows a diff of the policy bundle on commits to all other branches:
++
+[source,yaml]
+----
+version: 2.1
+
+orbs:
+  circleci-cli: circleci/circleci-cli@0.1.9 # Use orb to make the `circleci-cli/default` executor available for running jobs
+
+workflows:
+  main-workflow:
+    jobs:
+      - diff-policy-bundle:
+          context: <my-context>
+          filters:
+            branches:
+              ignore: main # on all branches other than main
+      - push-policy-bundle:
+          context: <my-context>
+          filters:
+            branches:
+              only: main # only on the main branch
+
+jobs:
+  diff-policy-bundle:
+    executor: circleci-cli/default
+    resource_class: small
+    steps:
+      - checkout
+      - run:
+          name: Diff policy bundle
+          command: circleci policy diff ./config --owner-id $ORG_ID # show a diff of the policy bundle
```

Comment:
@sagar-connect @davidmparrott where should this $ORG_ID env var be set?

Reply:
This org-id is not a secret like the circle-token. (as far as all users internal to an org).
It can even defined even withing the circle config of the policies repo too, or defined as a project environment variable.